lib/mock/store.dart

```diff
@@ -0,0 +1,36 @@
+typedef Data = Map<String, String>;
+typedef OneStore = List<Data>;
+typedef AllStore = Map<String, OneStore>;
+
+// todo: change fake data
```

Comment:
```suggestion
// TODO(Lorenzo): change fake data
```
You don't respect the flutter style
https://dart-lang.github.io/linter/lints/flutter_style_todos.html